### ISSUE TITLE
[BUG] config description will be lost when do clone config

### ISSUE THREAD

**Issue by xkcoding** [alibaba/nacos#1881]
## Issue Description

Type: *bug report*

### Describe what happened (or what feature you want)

config description will be lost when I clone config from one namespace to another namespace.

### Describe what you expected to happen

clone config and all the properties will be transferred from one namespace to another namespace.

### How to reproduce it (as minimally and precisely as possible)

1. clone one config from namespace A to namespace B.
2. the config description in namespace B will be lost.

### Tell us your environment

OS: Centos 7
Nacos: 1.1.3

### Anything else we need to know?

Nacos is not downloaded from the release page. I compile the code and run it.

**Comment by xkcoding** [comment]
I found that if want to fix this issue, it will modify the following points
- com.alibaba.nacos.config.server.controller.ConfigController#cloneConfig
```java
List<ConfigInfo> queryedDataList = persistService.findAllConfigInfo4Export(null,null, null, null, ids);
//  query ConfigInfo data list result type should be changed from ConfigInfo to ConfigAllInfo.
```
-  com.alibaba.nacos.config.server.service.PersistService#findAllConfigInfo4Export
```java
// query sql should be added config desc column in select  statement.
```
- com.alibaba.nacos.config.server.service.PersistService#batchInsertOrUpdate
```java
// shoud be support config desc field insert and update
```

when I reading the source code , and I find another bug in the code.

**Comment by xkcoding** [comment]
It has been fixed #1776
